t/05-scheduler-full.t

```diff
@@ -298,6 +299,28 @@ subtest 'Websocket server - close connection test' => sub {
       "Connection was turned off by ws server correctly";
 };
 
+
+subtest 'Worker logs correctly' => sub {
+    kill_service($wspid);
+    my $log_file = tempfile;
+    local $ENV{OPENQA_LOGFILE};
+    local $ENV{MOJO_LOG_LEVEL};
+    my $worker_pid = create_worker($k->key, $k->secret, "http://bogushost:999999", 1, $log_file);
```

Comment:
This was clever! :)

Reply:
Even tho, it should not be in scheduler-full test file :) -  i say this just because it is already a huge beast of test